SIT.Manager/Services/Install/InstallerService.cs

```diff
@@ -714,4 +716,34 @@ public async Task InstallSit(GithubRelease selectedVersion, string targetInstall
             throw;
         }
     }
+    public void CopyEftSettings(string targetInstallDir)
+    {
+        var sourcePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Battlestate Games", "Escape from Tarkov", "Settings");
+        var destinationPath = Path.Combine(targetInstallDir, "user", "sptSettings");
+
+        var filesToCopy = new[] { "Control.ini", "Game.ini", "Graphics.ini", "PostFx.ini", "Sound.ini" };
+        try
+        {
+            if (!Directory.Exists(destinationPath))
+            {
+                Directory.CreateDirectory(destinationPath);
+            }
+
+            foreach (var fileName in filesToCopy)
+            {
+                var sourceFile = Path.Combine(sourcePath, fileName);
+                if (File.Exists(sourceFile))
+                {
+                    var destFile = Path.Combine(destinationPath, fileName);
+                    File.Copy(sourceFile, destFile, overwrite: true);
+                }
+            }
+
+            _logger.LogInformation($"Successfully copied EFT settings to '{destinationPath}'.");
```

Comment:
IDE gets grumpy about string interpolation for logging. it expects `_logger.LogInformation("Successfully copied EFT settings to '{destinationPath}'.", destinationPath);`